adele-booking/src/main/java/com/microservicesteam/adele/booking/domain/TicketRepository.java

```diff
@@ -33,8 +33,9 @@ public Ticket get(Position position) {
         return tickets.get(position);
     }
 
-    ImmutableList<Ticket> getTicketsStatus() {
+    ImmutableList<Ticket> getTicketsStatus(long eventId) {
```

Comment:
Minor: I could read the code easier if the method would have a name like `getTicketsStatusByEvent`.